src/Nethermind/Nethermind.Runner/Program.cs

```diff
@@ -591,4 +586,32 @@ private static string GetProductInfo()
 
         return info.ToString();
     }
+
+    private static bool ValidateArguments(string[] args, CommandLineApplication app)
+    {
+        // Get all valid options from the configuration files
+        var validArguments = app.GetOptions().SelectMany(o => new[] { o.LongName, o.ShortName }).ToArray();
```

Comment:
avoid creating intermittent arrays of `new[] { o.LongName, o.ShortName }` with a simple loop.